arangod/Cluster/EnsureIndex.cpp

```diff
@@ -90,6 +90,12 @@ EnsureIndex::EnsureIndex(MaintenanceFeature& feature,
 
 EnsureIndex::~EnsureIndex() = default;
 
+// For local book keeping and reporting on /_admin/actions
+arangodb::Result EnsureIndex::setProgress(uint64_t u) {
+  _progress = u;
+  return arangodb::Result();
```

Comment:
Nit: can be simplified
```suggestion
  return {};
```